### PR TITLE
Search contact feature by Dikshya

### PR DESCRIPTION
**Pull request by Dikshyapoudel12**
# Description:
 This is a Gherkin file written for the Search Contact Feature.
## Issue:
https://github.com/SagarGi/FeatureFiles/issues/7

### CODE REVIEW

File: Features4/tests/feature/Search.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature:
```
Comment (SagarGi) [inline Features4/tests/feature/Search.feature]: rewrite this feature as told

File: Features4/tests/feature/Search.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    As a user
+    I want to search contacts
+    So that I can get contacts 
+
+Background:
+    Given I navigate to the login page
+    When I submit Email and password
+    And  Click on the "Login" button
+    Then I should be logged in 
+    Then Added Contact should be exists
```
Comment (kiranadh1452) [inline Features4/tests/feature/Search.feature]: Please reconsider this part.

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features4/tests/feature/Search.feature [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Feature: Search contact
+    As a user
+    I want to search contact by contact name
```
Comment (kiranadh1452) [inline Features4/tests/feature/Search.feature:3]:
```suggestion
    I want to search contact by name or email
```

File: Features4/tests/feature/Search.feature [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    So that only searched contact is displayed in contact list
+
+    Background:
+        Given the User has logged in with the valid email "deekshyap@gmail.com" and password "123456"
+        And  navigated to Contact Fox page
+        And the following contacts has been created
```
Comment (kiranadh1452) [inline Features4/tests/feature/Search.feature:9]:
```suggestion
        And the following contacts have been created 
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**